src/components/SearchBox.tsx

```diff
@@ -22,7 +22,7 @@ export const SearchBox = ({
           <SearchRoundedIcon
             fontSize="large"
             sx={ {
-              color: theme.palette.text.secondary,
```

Comment:
There were a couple of instances where we were using the `theme.palette` however it was not consistent throughout our app and the usages were minimal. I removed the palette where possible as i feel like that when we do decide to do theming and create a palette we can do a simple find and replace.